playground/package.json

```diff
@@ -3,7 +3,8 @@
   "type": "module",
   "private": true,
   "scripts": {
-    "dev": "vite",
+    "dev": "USE_DIRS=true vite",
+    "dev:presets": "vite",
     "build": "vite build",
```

Comment:
we should add a new `build:preset` script, and also add `USE_DIRS=true` to `build` script